wdl/transforms/new-base/src/main/scala/wdl/transforms/base/ast2wdlom/AstToCallElement.scala

```diff
@@ -30,8 +30,10 @@ object AstToCallElement {
 
     val callBodyValidation: ErrorOr[Option[CallBodyElement]] = ast.getAttributeAsOptional[CallBodyElement]("body").toValidated
 
-    (callableNameValidation, aliasValidation, afterValidation, callBodyValidation) mapN { (name, alias, after, body) =>
-      CallElement(name, alias, after, body)
+    val srcLocValidation : ErrorOr[Option[SourceFileLocation]] = ast.getSourceLine.map(SourceFileLocation.convert).validNel
```

Comment:
This doesn't look like it needs to be an `ErrorOr` since it's always using `.validNel`.

I'd just say `val srcLoc: Option[SourceFileLocation] = ast.getSourceLine.map(SourceFileLocation.convert)` and then you can use it directly in `CallElement(name, alias, after, body, srcLoc)` without even needing to make it an argument to the `mapN`